scripts/Adetoyebamise.js.txt

```diff
@@ -0,0 +1,11 @@
+const requirements ={
```

Comment:
You do not need to have two files of the same thing. Please delete this extra file.

Reply:
delete this file ade. thanks.
am passing